frontend/src/pages/AbsencePlanner.tsx

```diff
@@ -1,85 +1,77 @@
 import "../index.css";
-import React, { useState, forwardRef } from "react";
+import React, { useState } from "react";
 import { AuthContext } from "../components/AuthProvider";
 import { Toast } from "../components/Toast";
 import {
   ToastMsg,
   TimeEntry,
   IssueActivityPair,
   FetchedTimeEntry,
-  Issue,
   IdName,
+  AbsenceInterval,
 } from "../model";
 import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
-import sv from "date-fns/locale/sv";
 import {
   reportTime,
   dateFormat,
-  isWeekday,
   getTimeEntries,
-  getUsersInGroups,
-  getGroups,
+  getReportableWorkingDays,
+  areConsecutive,
+  getWeeksBetweenDates,
 } from "../utils";
-import { eachDayOfInterval, Interval, format as formatDate } from "date-fns";
+import {
+  format as formatDate,
+  isBefore,
+  isWithinInterval,
+  isAfter,
+  isSameDay,
+  startOfDay,
+  parseISO,
+  compareAsc,
+} from "date-fns";
+import sv from "date-fns/locale/sv";
 import LoadingOverlay from "react-loading-overlay-ts";
 import ClimbingBoxLoader from "react-spinners/ClimbingBoxLoader";
 import { HeaderUser } from "../components/HeaderUser";
-import { Chart } from "react-google-charts";
+
 import trash from "../icons/trash.svg";
 import pencil from "../icons/pencil.svg";
+
 import { useConfirm } from "../components/ConfirmDialogProvider";
 import { useSelectDates } from "../components/EditPeriodDialogProvider";
 import calender from "../icons/calendar-week-white.svg";
 
+import AbsenceIssuesSelector from "../components/AbsencePlanner/AbsenceIssuesSelector";
+
+export const absenceIssueOptions: { id: number; subject: string }[] = [
+  { id: 6992, subject: "Parental leave" },
+  { id: 6993, subject: "Sick leave" },
+  { id: 6994, subject: "VAB" },
+  { id: 6995, subject: "Vacation" },
+];
```

Comment:
Are these the only valid reasons? I would want to have an "Other" reason, and/or "Family matters" (typically a funeral or something that is not VAB or Parental leave).

Reply:
@viklund I think we need your input on this

Reply:
We have that, but then you just report it on a single day. Not required to use the long-term absence-planner/reporter in those instances.

Reply:
I would be willing to accept that, but the planner could be used for any future absence, even single days.

Reply:
Well, it's more like you might add VAB after the fact. It's now possible to report absence after the fact as well. But sure, we could add the "Other" reason here as well.

Reply:
Fixed!